opentelemetry-resourcedetector-gcp/src/opentelemetry/resourcedetector/gcp_resource_detector/_metadata.py

```diff
@@ -25,7 +25,8 @@
 _INSTANCE = "instance"
 _RECURSIVE_PARAMS = {"recursive": "true"}
 _GCP_METADATA_URL_HEADER = {"Metadata-Flavor": "Google"}
-_TIMEOUT_SEC = 5
+# Use a shorter timeout for connection so we won't block much if it's unreachable
+_TIMEOUT = (2, 5)
```

Comment:
This is decreasing the connect timeout and not read timeout IIUC https://requests.readthedocs.io/en/latest/user/advanced/#timeouts?

Is that what you meant?

Reply:
Yes, the use case is reducing the timeout if loading the resource detector outside GCP

Reply:
Makes sense, can you just update the PR description s/read/connect?

Reply:
Oh and also a changelog entry pls https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/blob/main/opentelemetry-resourcedetector-gcp/CHANGELOG.md

Reply:
Updated title, description and added CHANGELOG